src/ApparentHorizons/StrahlkorperDataBox.hpp

```diff
@@ -134,7 +134,7 @@ struct DxRadius : db::ComputeItemTag {
       typelist<Strahlkorper<Frame>, Radius<Frame>, InvJacobian<Frame>>;
 };
 
-/// d2x_radius(i,j) is \f$\partial^2 r/\partial x^i\partial x^j\f$.
+/// `D2xRadius`(i,j) is \f$\partial^2 r/\partial x^i\partial x^j\f$.
```

Comment:
Change class name to be consistent with derivative function names in scalar wave analytic solution